.standard_todo.yml

```diff
@@ -0,0 +1,169 @@
+# Auto generated files with errors to ignore.
+# Remove from this list as you refactor files.
+---
+ignore:
```

Comment:
What if we add blanket ignores for everything but profiling and use the `standard` rake task?

Thinking of it, it seems like it is the way to go as as is using standard on any other file (e.g my editor is configured to automatically run and fix using standard if it is available) will have standard lint it.

Reply:
It took me some fighting with standardrb but yeah I like this idea, so I've redone the integration using this approach in 6849df52f863e3a41240503fbf9289cd378ebba5 .